perfkitbenchmarker/traces/dstat.py

```diff
@@ -103,7 +103,7 @@ def _StopOnVm(self, vm):
     vm.RemoteCommand(cmd)
     try:
       vm.PullFile(self.output_directory, file_name)
-    except:
+    except Exception:
```

Comment:
Changed to Exception. I'm just trying to cut down on places where things like KeyboardInterrupt or SystemExit can be involuntarily swallowed.